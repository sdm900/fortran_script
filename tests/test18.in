do i=0,5
a(1,i+1) = 10.1+i
end do

do i=1,5
print a(1,i)
end do

a(0,-1) = 1
a(0,1) = 2

print a(1,3)
print a(0,-1)
print a(0,1)
print a(0,0)
