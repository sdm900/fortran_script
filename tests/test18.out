    10.10000 
    11.10000 
    12.10000 
    13.10000 
    14.10000 
    12.10000 
     1.00000 
     2.00000 
 15 Array value used before it is initialised
